EIPS/eip-2539.md

```diff
@@ -0,0 +1,266 @@
+---
+eip: 2539
+title: BLS12-377 curve operations
+author: Alex Vlasov (@shamatar)
+discussions-to: https://ethereum-magicians.org/t/eip-2539-bls12-377-precompile-discussion-thread/4659
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-02-26
+requires: 1109, 2046
+---
+
+## Simple Summary
+This precompile adds operation on BLS12-377 curve (from Zexe paper) as a precompile in a set necessary to *efficiently* perform operations such as BLS signature verification and perform SNARKs verifications. Unique properties of BLS12-377 also later allow to have SNARKs that check BLS12-377 pairing in an efficient way and allow e.g. constant-size BLS signature aggregation.
```

Comment:
Remove "from Zexe paper".  This is a technical document, not a place to give props/kudos/attribution.  If you think that the paper is particularly relevant, I recommend adding it to `../assets/eip-2539/zexe-paper.pdf` and linking to it in a place where it makes sense that a future reader of this specification would care.

Reply:
```suggestion
Adds BLS12-377 curve operations as precompiles.
```
Simple summary should be a very terse one-line description.  Longer than the title, but not meant to convey much information and instead let the reader quickly identify "am I in the right place?".  This may show up in a sub-heading somewhere for example.  The rest of this should probably go in the motivation, as it is is discussing *why* this is useful/interesting.